README.md

```diff
@@ -57,36 +51,47 @@ Package Name | Minimum Version
 
 Package Name | Minimum Version
 ------------ |:--------------:
-[sphinx](http://www.sphinx-doc.org/) | 1.4
-[inkscape](http://inkscape.org) |
+[`sphinx`](http://www.sphinx-doc.org/) | 1.8
 
-> **Note:** We recommend using [pip](https://pypi.org/project/pip) to manage Python modules.
-
-Python Module |
----------- |
-sphinx_rtd_theme |
-recommonmark |
+Python Module | Required version
+------------- |:---------------:
+`sphinx_rtd_theme` | 0.4.3
+`recommonmark` | 0.5.0
+`docutils` | 0.12
 
 
 ### Build process ###
 
-Gaffer also depends on a number of 3rd-party libraries and python modules, many of which are not entirely straightforward to build. We therefore recommend using the latest pre-built dependencies from the [Gaffer dependencies project](https://github.com/GafferHQ/dependencies/releases). These are used in our automated test builds and so are guaranteed to be up-to-date with Gaffer's requirements.
+```bash
+git clone https://github.com/GafferHQ/gaffer.git
+cd gaffer
+```
+
+Gaffer depends on a number of 3rd-party libraries. We recommend using the latest pre-built dependencies from the [Gaffer dependencies project](https://github.com/GafferHQ/dependencies/releases). These are used in our automated test builds and so are guaranteed to be up-to-date with Gaffer's requirements.
 
-Once you've downloaded the dependencies, unpack them and move/rename them to the directory in which you want to make your Gaffer build. We'll refer to this location as `<BUILD_DIR>` in the instructions below. Before continuing, make sure the dependencies are unpacked as `<BUILD_DIR>/include`, `<BUILD_DIR>/lib`, etc.
+The dependencies distribution forms the basis of a Gaffer build and the root of the expanded archive becomes the `BUILD_DIRECTORY` we pass to `scons`.
 
-Next, clone the Gaffer source and `cd` into its directory:
+The easiest way to get them is with the `config/installDependencies.sh` script included in the Gaffer source tree. This downloads the correct version for the version of Gaffer you are building and unpacks them to a directory of your choice.
+
+In this example we're going to build gaffer to a `gaffer-build` directory next to our checkout.
+
+```
+./config/installDependencies.sh ../gaffer-build
+```
+
+You can then build Gaffer itself:
 
 ```bash
-git clone https://github.com/GafferHQ/gaffer.git
-cd gaffer
+scons build BUILD_DIR=../gaffer-build
 ```
 
-You can then build Gaffer with `scons`:
+With any luck, you now have a functioning version of `gaffer`!
 
 ```bash
-scons BUILD_DIR=<BUILD_DIR> build
+../gaffer-build/bin/gaffer
 ```
 
+> NOTE: In order to build support for third-party renderers such as Arnold, you need to set `ARNOLD_ROOT` etc. options accordingly for your `scons` build. eg: `scons build ARNOLD_ROOT=/path/to/arnold/6 BUILD_DIR=...`
```

Comment:
Mentioning `ARNOLD_ROOT` for installs has been on my mind for a while. Since it's the most popular renderer, I think this should be a main piece of text, and not a note. Something like:

---

\#\#\# Building for third-party renderers
Gaffer dependencies ships with Appleseed, but to build the modules for one of the other supported third-party renderers, you will need to set an appropriate environment variable that points to its directory. The path environment variables are:

\- Arnold: \`ARNOLD_PATH\`
\- 3Delight: \`DELIGHT_PATH\`

For example, the following command builds Gaffer with the \`GafferArnold\` module:

\`\`\`bash
scons build ARNOLD_ROOT=/path/to/arnold/6 BUILD_DIR=...
\`\`\`

Reply:
Great idea, thanks. Improved in https://github.com/GafferHQ/gaffer/pull/3604/commits/8f41455423ba0863f98f69e61e5eb4a23bd03841